libraries/c_sdk/standard/ble/utest/ble_mqtt_transport/CMakeLists.txt

```diff
@@ -64,6 +66,8 @@
             "${AFR_ROOT_DIR}/freertos_kernel/include/"
             "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common/include"
             "${AFR_ROOT_DIR}/libraries/coreMQTT/source/include"
+            "${AFR_ROOT_DIR}/libraries/coreMQTT/test/unit-test"
```

Comment:
Why does this need to have the `coreMQTT/test/unit-test` path in the include list?